scripts/bench/tts-bench.py

```
@@ -0,0 +1,650 @@
+        # The release tarball ships its own onnxruntime and espeak-ng data next
+        # to the binary.
+        env["LD_LIBRARY_PATH"] = f"{self.binary.parent}:{env.get('LD_LIBRARY_PATH', '')}"
+        cmd = [
+            str(self.binary),
+            "-m",
+            str(self.voice_dir / f"{voice}.onnx"),
+            "-f",
+            str(out),
+        ]
+        start = time.monotonic()
+        proc = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)